Assets/Scripts/WorldProperties.cs

```diff
@@ -207,19 +314,44 @@ public static double UnityZToLong(double long1, double lat, float unityZCoord)
             return delLong;
         }
 
-        */
-        // Old logic to calculate new position of the drone and for waypoints.
-        /// Calculates the 3D displacement of the drone from it's initial position, to its current position, in Unity coordinates.
-        /*
-        changePos = new Vector3(
-            ((float) (WorldProperties.LatDiffMeters(InitialGPSLat, new_ROSPosition._lat)) / WorldProperties.Unity_X_To_Lat_Scale),
-                    ((new_ROSPosition._altitude - InitialGPSAlt) / WorldProperties.Unity_Y_To_Alt_Scale),
-                    ((float)(WorldProperties.LongDiffMeters(InitialGPSLong, new_ROSPosition._long, new_ROSPosition._lat) / WorldProperties.Unity_Z_To_Long_Scale))
-                  );
-        /// sets the drone Game Object's local position in the Unity world to be it's start position plus the newly calculated 3d displacement to the drone's current position.
-        // Peru 6/9/20: Phasing out World Properties variables used in depreciated script
-        // drone.transform.localPosition = WorldProperties.selectedDroneStartPos + offsetPos + changePos;
-        */
+        /// <summary>
```

Comment:
Which files still use these properties? I think they are outdated calculations...

Reply:
yep, I'll delete the comments